.github/workflows/swift-toolchain.yml

```diff
@@ -793,12 +793,32 @@ jobs:
           path: |
             ${{ github.workspace }}/BinaryCache/1/**/*.pdb
 
-      - uses: microsoft/action-publish-symbols@v2.1.6
+      - name: Upload PDBs to Azure
+        uses: microsoft/action-publish-symbols@v2.1.6
         if: ${{ needs.context.outputs.debug_info }}
         with:
           accountName: ${{ vars.SYMBOL_SERVER_ACCOUNT }}
           personalAccessToken: ${{ secrets.SYMBOL_SERVER_PAT }}
           symbolsFolder: ${{ github.workspace }}/BinaryCache/1
+          searchPattern: '**/bin/**/*.pdb'
```

Comment:
@compnerd I made these explicit (from the action's default). Is this pattern correct, and should I follow the same format for dlls and exes? I don't have a toolchain build handy to check.

Reply:
Turns out `dispatch.dll`/`dispatch.pdb` are not emitted within a "bin" directory, so we are not uploading them. (fyi @hjyamauchi )

Reply:
Yeah, libdispatch not using `bin` for the output is why the default pattern didn't work.